service/src/test/java/bio/terra/workspace/common/fixtures/ControlledAwsResourceFixtures.java

```diff
@@ -74,14 +84,29 @@ public static ControlledAwsS3StorageFolderResource.Builder makeAwsS3StorageFolde
 
   public static ControlledAwsSageMakerNotebookResource makeDefaultAwsSagemakerNotebookResource(
       UUID workspaceUuid) {
+    return makeAwsSageMakerNotebookResourceBuilder(
+            workspaceUuid, "sagemaker-resource", "foo-instance")
+        .build();
+  }
+
+  public static ControlledAwsSageMakerNotebookResource.Builder
+      makeAwsSageMakerNotebookResourceBuilder(String instanceName) {
+    return makeAwsSageMakerNotebookResourceBuilder(
+        WORKSPACE_ID, "sagemaker-resource", instanceName);
```

Comment:
nit: since resource names need to be unique within a workspace, it might be helpful to append random numbers here. Otherwise, callers will need to override this when creating multiple resources

Reply:
done